packages/ndla-ui/src/Grid/Grid.tsx

```diff
@@ -7,43 +7,70 @@
  */
 
 import styled from '@emotion/styled';
-import { breakpoints, colors, misc, mq, spacing } from '@ndla/core';
-import { ReactNode } from 'react';
+import { breakpoints, colors, misc, mq, spacing, spacingUnit } from '@ndla/core';
+import { HTMLAttributes, ReactNode } from 'react';
 
-export interface GridProps {
-  columns: 2 | 4;
+export interface GridProps extends HTMLAttributes<HTMLDivElement> {
+  columns: '2' | '4' | '2x2';
```

Comment:
Er det meiniga at du skal kunne velge 2x2 i ed? Eg trudde 4 skulle breake til 2x2 ved små skjermer. Ser forresten at Hedvig forventa at den skulle gå til 3x1 før 2x2 ved gradvis mindre skjermer.

Reply:
Vi må ha 2x2 for å støtte omNdla sitt design. Der skal nøkkeltall være 2x2 hele tiden. Det at det skal breake til 3x1 vil bli vanskelig, jeg kan å høre ved henne imorgen